### DISCUSSION TITLE
Feature: No Reload

### DISCUSSION THREAD

**Discussion by dwsiddall** [YimMenu/HorseMenu#120] [Ideas]
A favorite feature goes in conjunction with Infinite Ammo, "No Reload"...so then gun just shoots continuously without reloading.

**Comment by Rxann** [comment]
That is a thing already. Infinite Clip.

**Reply by dwsiddall** [reply]
Perfect.  I was confused by the word clip because it was used elsewhere to mean moving freely unconstrained..

**Reply by Rxann** [reply]
Yes, No Clip is like you can go anywhere
